src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -228,7 +228,16 @@ public void TestStartOnUnixWithBadPermissions()
         {
             string path = GetTestFilePath();
             File.Create(path).Dispose();
-            Assert.Equal(0, chmod(path, 644)); // no execute permissions
+            int mode;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("FREEBSD")))
```

Comment:
Can you lift this into eg PlatformDetection.IsFreeBsd?

Reply:
I'll take a look @danmosemsft . Thanks for looking. This is preliminary change.